### PR TITLE
Use Fedora/RHEL names for RPM deps

### LINKED ISSUE #2950: Configurator on Fedora 36 Workstation
### Describe the bug

I'm tried to install BetaFlight configurator 10.8.0 on Fedora 36, but it have a dependency unsolved: libgconf-2-4

### To Reproduce

zandor@tiger Downloads]$ sudo dnf install ./betaflight-configurator-10.8.0-1.x86_64.rpm 
[sudo] password for zandor: 
Last metadata expiration check: 0:02:44 ago on Tue 14 Jun 2022 11:00:49 PM -03.
Error: 
 Problem: conflicting requests
  - nothing provides libgconf-2-4 needed by betaflight-configurator-10.8.0-1.x86_64
(try to add '--skip-broken' to skip uninstallable packages)

### Expected behavior

Successful instalation

### Configurator version

10.8.0

### Flight controller configuration

Do not apply

### Add any other context about the problem that you think might be relevant here

_No response_

### PR DESCRIPTION
**Pull request by kevinsumner**
Fixes #2950

### CODE REVIEW

**Comment by kevinsumner** [comment]
I'm not super familiar with npm-based development, but I was able to `gulp release`, `dnf install ./release/betaflight-configurator-10.8.0-1.x86_64.rpm`, then run Betaflight Configurator successfully. I verified git commit hash in Configurator matched the this commit.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by McGiverGim — APPROVED** [review]
It seems ok to me. Thanks!

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**
